Dockerfile

```diff
@@ -1,22 +1,29 @@
 # build environment
 FROM node:10.8.0 as builder
+
 RUN mkdir -p /usr/omniboard
 WORKDIR /usr/omniboard
-ENV PATH /usr/omniboard/node_modules/.bin:$PATH
-
-# grab tini for signal processing and zombie killing
-ENV TINI_VERSION v0.18.0
```

Comment:
Why was tini removed?

Reply:
From docker api v1.25+, docker containers can use the built in init flag https://docs.docker.com/engine/reference/commandline/run/ https://docs.docker.com/config/containers/multi-service_container/

Of course, if you want to keep tini (since it's a run flag, and not a build flag) I can add tini back, you decide :)

Reply:
Yes, let's bring back tini